src/styles.css

```diff
@@ -1 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
+ "styles": [
```

Comment:
this should be like https://github.com/inspirehep/record-editor/blob/master/angular-cli.json#L22